frontend/src/Pages/Admins.tsx

```diff
@@ -22,7 +22,7 @@ import { useDebounceValue } from 'usehooks-ts';
 import Pagination from '@/Components/Pagination';
 import API from '@/api/api';
 
-export default function Users() {
+export default function Students() {
```

Comment:
This shouldn't be students- it should be Admin or AdminManagement to make it more specific to what the page is actually doing.